wagtail/wagtailcore/models.py

```diff
@@ -1035,6 +1035,14 @@ def get_verbose_name(cls):
         # except this doesn't convert any characters to lowercase
         return capfirst(cls._meta.verbose_name)
 
+    @classmethod
+    def get_help_text(cls):
+        """
+        Returns human-readable help text used to describe this page model.
+        If help text is not found, default to verbose name.
+        """
+        return getattr(cls, 'page_help_text', cls.get_verbose_name())
```

Comment:
If we decided to implement [the way I suggested](https://github.com/wagtail/wagtail/pull/3699#discussion_r142094340), we will not need fallback to verbose name. We will just show the help text, if present.